{"name": "test_login[data0]", "status": "failed", "statusDetails": {"message": "AssertionError: 配置文件 config/config.yaml 读取失败，请检查路径或内容\nassert None is not None", "trace": "@pytest.fixture(scope=\"session\", autouse=True)\n    def init_token():\n        \"\"\"初始化环境配置\"\"\"\n        cfg = read_yaml(\"config/config.yaml\")\n>       assert cfg is not None, \"配置文件 config/config.yaml 读取失败，请检查路径或内容\"\nE       AssertionError: 配置文件 config/config.yaml 读取失败，请检查路径或内容\nE       assert None is not None\n\nconftest.py:9: AssertionError"}, "description": "\n    用户登录测试\n    ", "parameters": [{"name": "data", "value": "{'name': 'test001', 'password': '123456'}"}], "start": 1761027515550, "stop": 1761027515550, "uuid": "c2fb8001-0689-48b1-9657-95e528761c72", "historyId": "1a7eeada76a8eddd82d6ce3b0e63a4d6", "testCaseId": "ab4ae781b39039ec4eae113a075c2b68", "fullName": "testcases.test_user_login#test_login", "labels": [{"name": "parentSuite", "value": "testcases"}, {"name": "suite", "value": "test_user_login"}, {"name": "host", "value": "DESKTOP-UGQEI6T"}, {"name": "thread", "value": "11844-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.test_user_login"}], "titlePath": ["testcases", "test_user_login.py"]}